src/layer.rs

```diff
@@ -24,7 +24,6 @@ use tracing_subscriber::{
 
 /// A tracing adapater for stackdriver
 pub struct Stackdriver<W = fn() -> io::Stdout> {
-    time: DateTime<Utc>,
```

Comment:
atm, i am not seeing a need for time field. i guess what we would need is a format field ?